prime-router/docs/openapi.yml

```diff
@@ -1,4 +1,28 @@
 openapi: 3.0.2
+# ,---. .                                           .               
```

Comment:
Lovely!  Its also ok to just delete the file; we can always retrieve it from github if needed.

Reply:
I'd like to combine removal with an update to ReportStream-Programmers-Guide-v2.0.docx.